pkg/e2e/e2e.go

```diff
@@ -132,13 +132,18 @@ func beforeSuite() bool {
 		if err != nil {
 			log.Printf("Error generating Kube Clientset: %v\n", err)
 		}
-		ctx, cancel := context.WithTimeout(context.Background(), time.Hour*2)
+		duration, err := time.ParseDuration(viper.GetString(config.Tests.ClusterHealthChecksTimeout))
+		if err != nil {
+			log.Printf("Failed parsing health check timeout, using 2 hours: %v", err)
+			duration = time.Hour * 2
```

Comment:
Re-defaulting this will mask certain configuration and coding errors. IMHO it would be better to just blow up.

Reply:
Fair enough, It is done.